enterprise/admin/forms.py

```diff
@@ -61,15 +67,25 @@ class ManageLearnersForm(forms.Form):
             "by the heading 'email' in the first row. Optionally, the .csv file may contain "
             "a column of course run keys, indicated by the heading 'course_id' in the first row, to "
             "enroll learners in multiple courses."
-        )
+        ),
     )
     course = forms.CharField(
-        label=_("Enroll these learners in this course"), required=False,
+        label=_("Enroll these learners in this course"),
+        required=False,
         help_text=_("To enroll learners in a course, enter a course ID."),
     )
+    force_enrollment = forms.BooleanField(
+        label=_("Force Enrollment"),
+        help_text=_(
+            "The selected course is 'Invite Only'. Only staff can enroll learners to this course."
+        ),
+        required=False,
+    )
     course_mode = forms.ChoiceField(
-        label=_("Course enrollment track"), required=False,
-        choices=BLANK_CHOICE_DASH + [
+        label=_("Course enrollment track"),
+        required=False,
+        choices=BLANK_CHOICE_DASH
+        + [
```

Comment:
@tecoholic, many changes in this PR look like an automatic run of a linting tool. Is there a reason to include them here? They do not match the current state of the `master` branch; they are also not present in the PR we're backporting. I like that we're improving the code quality, but it makes diffs harder to compare directly. If we want to do this, we should open an upstream PR first.

Could we remove these additional changes and stick to backporting only the code that was added in https://github.com/openedx/edx-enterprise/pull/1748? If there are other changes made in the meantime that are needed for this to work, we should include them in a separate commit (preferably placed before this one).

Reply:
@Agrendalath oops.. Thanks for noticing this. Yes, these changes are definitely due to some automated formatting tool. I seem to have run something by accident. Github had hidden the diff as it's large and I must have missed it.

Reply:
I have updated the forms.py file with only necessary changes.